WordPress/Classes/Extensions/Math.swift

```diff
@@ -22,14 +22,13 @@ extension Int {
 }
 
 extension Comparable {
-    /**
-     Clamps self between a minimum and maximum value
-     
-     - returns: the method returns
-        - min if self < min
-        - max if self > max
-        - otherwise it returns self
-     */
+    /// Clamps self between a minimum and maximum value
+    ///
+    /// - Returns: the method returns
+    ///     - min if self < min
+    ///     - max if self > max
+    ///     - otherwise it returns self
+    ///
```

Comment:
In this PR i ended up following what was found in most places (the white line before the method / property prettifies things).

Definitely, after skimming thru all of our Swift basecode... i couldn't agree more. We *need* a style guide, ASAP. Multiple inconsistencies everywhere!!